### PR TITLE
UR Beta Banner - Feedback page

### PR DESCRIPTION
**Pull request by EdwinKruglov**
## Changes in this PR

- add user feedback page, model and form

## Screen-shots or screen-capture of UI changes

https://user-images.githubusercontent.com/12530193/154666209-80dafbc3-937d-40ac-9ee7-9e2b9d7c12ac.mov

### CODE REVIEW

**Comment by EdwinKruglov** [comment]
[UR Beta Banner - Feedback page](https://trello.com/c/HfiVgDzW/279-ur-beta-banner-feedback-page)

File: spec/features/self-serve/feedback/feedback_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+RSpec.feature "User feedback page" do
```
Comment (peterdavidhamilton) [inline spec/features/self-serve/feedback/feedback_spec.rb:1]: Really nice @EdwinKruglov, could you run this feature spec with documentation output  and post it here please

Comment (EdwinKruglov) [reply spec/features/self-serve/feedback/feedback_spec.rb:1]: image

Comment (peterdavidhamilton) [reply spec/features/self-serve/feedback/feedback_spec.rb:1]: Please could you share this with a few people familiar with the ticket and get their take on the wording. Could it be improved at all?

- does it describe the feature like a story
- does it describe the criteria being tested
- does it make sense to non-devs
- does it explain a "feature" in business terms
- do the person, tense and grammar match

**Review by peterdavidhamilton — APPROVED** [review]

File: app/views/feedback/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+            <h1 class="govuk-fieldset__heading"><%= I18n.t("feedback.survey.title") %></h1>
+          </legend>
+          
+          <%= form.govuk_radio_buttons_fieldset :satisfaction, legend: { text: I18n.t("feedback.survey.satisfaction_question"), size: "s" } do %>
+              <% @satisfaction_options.each do |option| %>
+                <%= form.govuk_radio_button :satisfaction, option, label: { text: I18n.t("feedback.survey.satisfaction_options.#{option}") } %>
```
Comment (shafiqahmedkhan) [inline app/views/feedback/new.html.erb:18]: test

**Review by shafiqahmedkhan — APPROVED** [review]

**Pull request merged**